.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ jobs:
     <<: *defaults-release
     steps:
       - checkout
-      - run: pyenv global 3.7.0
+      - run: pyenv global 3.10.2
```

Comment:
Given it's just to trigger `dennis` I didn't want to bother with installing a specific version - this is just whatever the `ubuntu-2004:202201-02` has by default for Python 3.